src/devices/Tm1637/Display.cs

```diff
@@ -0,0 +1,35 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Iot.Device.Tm1637
+{
+    /// <summary>
+    /// Pre build characters that can be displayed
+    /// </summary>
+    public enum Display
+    {
+        Car0 = 0,
```

Comment:
Should this be `Char0` (and similarly below?)

Reply:
yes, will fix.